examples/asr/conf/speech_translation/fast-conformer_transformer.yaml

```diff
@@ -176,7 +176,7 @@ model:
       min_lr: 1e-6
 
 trainer:
```

Comment:
this config should be copied for the new `EncDecMultiTaskModel` model class: `conf/speech_multitask/fast-conformer_aed.yaml`